src/app/index.html

```diff
@@ -15,7 +15,8 @@
   pageTitle="Welcome"
   layout="container"
   showBreadcrumbs="true"
-  showTableOfContents="true">
+  showTableOfContents="true"
+  #stache>
```

Comment:
Also, we could add a plugin to add the `#stache` attribute to the `<stache>` component tags if that's a concern.